documentation/meta/monitoring/cloudwatch_logs/index.md

```diff
@@ -0,0 +1,251 @@
+2023-06-04T00:32:12.541Z	return func(*args, params=params, headers=headers, **kwargs)
+2023-06-04T00:32:12.541Z	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2023-06-04T00:32:12.541Z	File "/venv/lib/python3.11/site-packages/elasticsearch/client/__init__.py", line 1467, in reindex
+2023-06-04T00:32:12.541Z	return self.transport.perform_request(
+2023-06-04T00:32:12.541Z	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2023-06-04T00:32:12.541Z	File "/venv/lib/python3.11/site-packages/elasticsearch/transport.py", line 466, in perform_request
+2023-06-04T00:32:12.541Z	raise e
+2023-06-04T00:32:12.541Z	File "/venv/lib/python3.11/site-packages/elasticsearch/transport.py", line 427, in perform_request
+2023-06-04T00:32:12.541Z	status, headers_response, data = connection.perform_request(
+2023-06-04T00:32:12.541Z	^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2023-06-04T00:32:12.541Z	File "/venv/lib/python3.11/site-packages/elasticsearch/connection/http_requests.py", line 216, in perform_request
+2023-06-04T00:32:12.541Z	self._raise_error(response.status_code, raw_data)
+2023-06-04T00:32:12.541Z	File "/venv/lib/python3.11/site-packages/elasticsearch/connection/base.py", line 328, in _raise_error
+2023-06-04T00:32:12.541Z	raise HTTP_EXCEPTIONS.get(status_code, TransportError)(
+2023-06-04T00:32:12.541Z	elasticsearch.exceptions.RequestError: RequestError(400, 'search_phase_execution_exception', 'too_many_clauses: maxClauseCount is set to 1024')
+```
+
+Each line (prepended with a timestamp), is a separate log event. The implication
+of this is that if you're querying for a particular phrase that appears in a
+stack trace, you will only get the exact line of the stack trace with the phrase
+you queried for.
+
+This will be fixed generally for our Python applications, but still keep this in
+mind if you find logs that appear to be incomplete. It may be that the event
+parsing configuration needs tweaking to account for certain edge cases. Please
+open an issue if you notice this.
+
+### Logs Insights queries are not free
+
+Logs Insights queries parse logs on-the-fly (more or less) and cost money each
+time they are executed. You can minimise costs in the following ways:
+
+- Scope queries to as specific a period of time as possible. If you are
+  investigating an incident with a known beginning and end, narrow the query to
+  that time period via the date range selector at the top. If there are multiple
+  periods with lulls in between, consider performing multiple queries and
+  aggregating them manually outside Logs Insights.
+- Develop queries against the minimum required number. While you're still
+  building a query to find specific data, narrow the time range or use `limit`
+  to reduce the number of log lines processed during each iteration. Avoid
+  searching the full relevant data set until you're confident that the query
+  works and extracts the data you expect.
+- When trying to find examples of specific queries, use `limit` with a very low
```

Comment:
```suggestion
- When trying to find examples of specific logs, use `limit` with a very low
```
I think I know what you mean, but given the previous section talks about "example queries" it reads as though what you're looking for is queries to use as an example. Maybe `example logs matching a specific query`?

Reply:
Whoops, yes I meant logs here, thanks.